### PR TITLE
Add SHL manager URL config

### PR DESCRIPTION
**Pull request by ivan-c**
- Add SHL manager URL config to cpro

### CODE REVIEW

**Comment by ivan-c** [comment]
@daniellrgn

if there's a specific page that you'd like the user to land on (coming from cpro), can you please add it as a suggestion? thanks!

File: dev/docker-compose.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ services:
       SERVER_NAME: cpro.${BASE_DOMAIN}
       HTTPS: "true"
       INSTANCE_ID: inform
+      SHL_MANAGER_URL: https://shl-creator.${BASE_DOMAIN}
```
Comment (daniellrgn) [inline dev/docker-compose.yaml:11]:
```suggestion
      SHL_MANAGER_URL: https://shl-creator.${BASE_DOMAIN}/share
```

**Review by daniellrgn — APPROVED** [review]
@ivan-c Added the manager endpoint

**Review by achen2401 — APPROVED** [review]

File: dev/docker-compose.yaml (hunk 1 above)
Comment (ivan-c) [reply dev/docker-compose.yaml]: thanks!

**Pull request merged**